src/swbf_fnv_hashes.cpp

```diff
@@ -1,2834 +1,3935 @@
 
 #include "swbf_fnv_hashes.hpp"
 #include "synced_cout.hpp"
-
 #include <cstdint>
+#include <fstream>
+#include <sstream>
+#include <iostream>
+#include <mutex>
+#include <regex>
 #include <string>
 #include <unordered_map>
 #include <utility>
+#include <vector>
 
 using namespace std::literals;
 
-namespace {
+std::unordered_map<std::uint32_t, std::string> swbf_hashes;
```

Comment:
I was having real trouble with the initialization once the unordered_map was no  longer const.
If you can give an example of proper syntax for this, I'll incorporate it.

Reply:
That is strange, it works for me just to remove `const` from it. Though thinking about it the change to storing strings rather than string views would require editing the `_fnvp` operator as well, could that be it? This below snippet worked for me locally.
```c++
std::pair<std::uint32_t, std::string> operator""_fnvp(const char* str, std::size_t length)
{
   return {fnv_1a_hash({str, length}), {str, length}};
}

std::unordered_map<std::uint32_t, std::string> swbf_hashes{ // etc...
```